docs/source/contributing/developing.adoc

```diff
@@ -194,11 +193,11 @@ $go test -v <relative path of package> -run "Test<Regex pattern to match tests>"
 For more information about test options, run the `go test --help` command and review the documentation.
 
 [[integration-tests]]
-=== Running integration tests
+=== Running Integration Tests
 
-Integration tests utilize https://github.com/DATA-DOG/godog[`godog`], which uses Gherkin (Cucumber) to define test cases.
-The test cases are defined in *_test/integration/+*+.feature_*.
-Per default, the tests are executed against the binary created by `make build`, that is *_$GOPATH/bin/minishift_*.
+Integration tests utilize link:https://github.com/DATA-DOG/godog[Godog], which uses Gherkin (Cucumber) to define test cases.
+The test cases are defined in *_test/integration/features_*.
```

Comment:
That's not correct anymore. Why did you change this? There is no directory _test/integration/features_, there is a __test/integration_ and it contains several files with the feature suffix. This is what was expressed in the existing expression. Also, it needs the extra '+' since one needs to escape the '*'. Let's change this back please.